dynomitemanager/src/main/java/com/netflix/dynomitemanager/defaultimpl/DynomitemanagerConfiguration.java

```diff
@@ -704,11 +713,23 @@ public String getCommaSeparatedCassandraHostNames() {
     public boolean isEurekaHostSupplierEnabled() {
 	return configSource.get(CONFIG_IS_EUREKA_HOST_SUPPLIER_ENABLED, DEFAULT_IS_EUREKA_HOST_SUPPLIER_ENABLED);
     }
-    
+
     // Redis compatibility
     @Override
     public String getRedisCompatibleEngine() {
 	return configSource.get(CONFIG_DYNO_REDIS_COMPATIBLE_ENGINE, DEFAULT_REDIS_COMPATIBLE_ENGINE);
     }
-    
+
+    // Redis
+    // =====
+
+    /**
+    * Get the full path to the redis.conf configuration file.
+    * @return the {@link String} full path to the redis.conf configuration file
+    */
+    @Override
+    public String getRedisConf() {
```

Comment:
To be honest there are other Redis related things that needs to move to the `RedisStorageProxy`. This will allow an easy way to switch storage engines.